aten/src/ATen/detail/CUDAHooksInterface.h

```diff
@@ -133,6 +133,10 @@ struct CAFFE2_API CUDAHooksInterface {
     return false;
   }
 
+  virtual bool hasTensorCore() const {
```

Comment:
I'm not sure you want virtual function on a hot path - can you maybe do it in cudnn/Conv.cpp, where the backend is already known?